devicon.json

```
@@ -6122,6 +6122,26 @@
         "color": "#00b4ff",
         "aliases": []
     },
+    {
+        "name": "qwik",
```

The altnames attribute is missing.
If there are no alternative names, you can just have an empty array (see suggestion 👇)

```suggestion
        "name": "qwik",
        "altnames": [],
```